polkadot/node/core/candidate-validation/src/tests.rs

```diff
@@ -617,7 +543,7 @@ fn candidate_validation_multiple_ambiguous_errors_is_invalid() {
 
 // Test that we retry on internal errors.
 #[test]
-fn candidate_validation_retry_internal_errors() {
+fn candidate_validation_dont_retry_internal_errors() {
```

Comment:
Can you please refactor this into two test cases, backing and approval, making sure that one retries and one doesn't.